src/main/resources/META-INF/native-image/native-image.properties

```diff
@@ -1,8 +1,10 @@
 Args = --no-fallback \
        --install-exit-handlers \
        -H:+ReportExceptionStackTraces \
-       -H:+UnlockExperimentalVMOptions \
        -O1 \
        --initialize-at-build-time=org.fusesource.jansi \
        --initialize-at-run-time=org.fusesource.jansi.internal.JansiLoader \
        -H:IncludeResources=org/fusesource/jansi/internal/native/.*libjansi\.(jnilib|so)
+
+JavaArgs = --enable-native-access=ALL-UNNAMED \
+           --sun-misc-unsafe-memory-access=allow
```

Comment:
Without these we get some annoying warnings (same thing elsewhere where you see these options). They're from our dependencies so I think we have to live with this.